packages/edit-site/src/components/global-styles/screen-revisions/use-get-global-styles-revisions.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { useContext, useMemo } from '@wordpress/element';
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../../private-apis';
+
+const SITE_EDITOR_AUTHORS_QUERY = {
+	per_page: -1,
+	_fields: 'id,name,avatar_urls',
+	context: 'view',
+	capabilities: [ 'edit_theme_options' ],
+};
+
+const { GlobalStylesContext } = unlock( blockEditorPrivateApis );
+export default function useGetGlobalStylesRevisions() {
```

Comment:
Nit but `useGetGlobalStylesRevisions` makes me think the hook will return a function named `getGlobalStylesRevisions`, which it does not. I'd name it `useGlobalStylesRevisions`.